streams/src/main/java/org/apache/kafka/streams/kstream/internals/FullChangeSerde.java

```diff
@@ -21,19 +21,15 @@
 import org.apache.kafka.common.serialization.Serializer;
 
 import java.nio.ByteBuffer;
-import java.util.Map;
 
 import static java.util.Objects.requireNonNull;
 
-public final class FullChangeSerde<T> implements Serde<Change<T>> {
+public final class FullChangeSerde<T> {
```

Comment:
Since the buffer is no longer transparent to the value type, we actually don't need this to be a "real" serde anymore, and moving away from it liberates us to add more detailed manipulations (e.g., splitting the array into two arrays without deserializing).